assets/scripts/index.js

```diff
@@ -0,0 +1,88 @@
+const retrieveBooks = () => {
+  const bookList = document.querySelector('.books-list');
+  if (localStorage.getItem('books')) {
+    const books = JSON.parse(localStorage.getItem('books'))
+      .map(
+        (book) => `<li class="book-item" data-id="${book.id}">
+        <div class="book-item__info">
+          <h2 class="book-item__title">${book.title}</h2>
+          <p class="book-item__author">${book.author}</p>
+          <button class="removeFromShelfBtn" onclick="removeBook(${book.id})">
+            Remove
+          </button>
+        </div>  </li>`,
+      )
+      .join('');
+    bookList.innerHTML = books;
+  } else {
+    console.log('No books found');
+  }
+};
+
+window.onload = () => {
+  retrieveBooks();
+};
+const booksArray = [];
```

Comment:
- [ ] Kindly be noticed that you should make get the data from the local storage and put it in the booksArray so when you add data to booksArray it should append to the data in the local storage ie: 
`const booksArray = JSON.parse(localStorage.getItem('books'))? JSON.parse(localStorage.getItem('books')) : [];`